src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/gerritnotifier/GerritNotifier.java

```diff
@@ -68,6 +73,34 @@ public GerritNotifier(IGerritHudsonTriggerConfig config, GerritCmdRunner cmdRunn
         this.parameterExpander = new ParameterExpander(config, jenkins);
     }
 
+    /**
+     * Send Notification to gerrit.
+     *
+     * @param notification The notifications.
+     */
+    private void send(INotification notification) {
+
+        if (notification == null) {
+            logger.error("NotificationObject is null not sending command!");
+            return;
+        }
+
+        if (!notification.isValid()) {
+            logger.error("Something wrong during parameter extraction. "
+                    + "Gerrit will not be notified of BuildStarted!");
+            return;
+        }
+
+        NotificationCommands notifyCommands = notification.getCommands();
+        cmdRunner.sendCommand(notifyCommands.getCommand());
+
+        if (notification.isVoteSameTopic() && notifyCommands.hasTopicChanges()) {
```

Comment:
I wonder if this conditional is necessary.  If it was false, shouldn't notifyCommands.getCommandsTopicChanges() be an empty list?  Requiring the conditional put a burden on the caller to understand a bit about how topics are handled.

Reply:
That's right. Removed the additional check and only kept checking if `isVoteSameTopic()` is enabled.